### PR TITLE
Fix:  <AyushJain001>

### LINKED ISSUE #1606: [PROFILE ADDITION] <Ayush Jain>
### Profile screenshot

image

### PR DESCRIPTION
**Pull request by AyushJain001**
resolve #1606

### CODE REVIEW

File: profiles/AyushJain001.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/90152799/210181328-0922e8d4-bea4-4643-b1d3-186250b87d94.png",
```
Comment (Panquesito7) [inline profiles/AyushJain001.json:2]: I recommend more of an image like this. What do you think? 🙂

image

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/210187235-4d9af402-a934-4622-8e68-44a56d4a47c8.png",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by AyushJain001** [comment]
Okay I am changing the image

**Comment by AyushJain001** [comment]
Updated the requested changes. Kindly check.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**